py-polars/polars/lazyframe/frame.py

```diff
@@ -3235,13 +3216,15 @@ def group_by_dynamic(
         ...         "A": ["A", "A", "B", "B", "B", "C"],
         ...     }
         ... )
-        >>> lf.group_by_dynamic(
-        ...     "idx",
-        ...     every="2i",
-        ...     period="3i",
-        ...     include_boundaries=True,
-        ...     closed="right",
-        ... ).agg(pl.col("A").alias("A_agg_list")).collect()
+        >>> (
+        ...     lf.group_by_dynamic(
+        ...         "idx",
+        ...         every="2i",
+        ...         period="3i",
+        ...         include_boundaries=True,
+        ...         closed="right",
+        ...     ).agg(pl.col("A").alias("A_agg_list"))
+        ... ).collect()
```

Comment:
Looks like something messed up the formatting here?

Reply:
yup, thanks 👍